src/content_script/PopupCard.tsx

```diff
@@ -24,7 +24,8 @@ import { ErrorBoundary } from 'react-error-boundary'
 import { ErrorFallback } from '../components/ErrorFallback'
 import { getBrowser, getSettings, isDesktopApp, ISettings } from '../common/utils'
 import { Settings } from '../popup/Settings'
-import { calculateMaxTop } from '.'
+import { calculateMaxXY } from '.'
```

Comment:
We cannot import anything from index.tsx in PopupCard because it will cause nested rendering issues in Tauri applications. In the future, we will consider moving PopupCard to a suitable location to prevent this situation from being accidentally introduced.

https://github.com/yetone/openai-translator/issues/226

Reply:
Okay, this function calculates the position of `PopupCard`, I think we can temporarily move to `src/content_script/utils.ts`, what do you think?

Reply:
@imyuanx Great idea. I already moved `calculateMaxTop` to the utils yesterday.

Reply:
I see. Now the conflict has been resolved!